packages/lesswrong/components/common/SearchBar.tsx

```diff
@@ -206,7 +206,7 @@ class SearchBar extends Component<SearchBarProps,SearchBarState> {
             {alignmentForum && <VirtualMenu attribute="af" defaultRefinement="true" />}
             <div onClick={this.handleSearchTap}>
               <SearchIcon className={classes.searchIcon}/>
-              { inputOpen && <SearchBox reset={null} focusShortcuts={[]} autoFocus={true} /> }
+              { inputOpen && <SearchBox reset={undefined} focusShortcuts={[]} autoFocus={true} /> }
```

Comment:
This breaks the settings reset button. I intentionally put null there because that was the only thing that actually made the reset button disappear. See this screenshot for what happens on this current PR: 

![](https://user-images.githubusercontent.com/9090789/81730285-2653cd00-9442-11ea-9b14-4657b29ca354.png)

Reply:
Correct fix is probably to add a comment to this effect and then add a ts-ignore.